core/server/worker/src/main/java/alluxio/worker/netty/ReadRequest.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.worker.netty;
+
+import alluxio.util.IdUtils;
+
+import javax.annotation.concurrent.ThreadSafe;
+
+/**
+ * Represents a read request received from netty channel.
+ */
+@ThreadSafe
+class ReadRequest {
+  private final long mId;
```

Comment:
I was using the previous field name. Currently, for `WriteRequest`, it can be block ID or temp file ID in UFS. To be in parallel, I use `mId` in `ReadRequest` too